src/pages/model.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import App from '../../scripts/flux/components/app'
+
+export default React.createClass({
+  displayName: 'ModelPage',
+  render () {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).